src/test/java/org/icgc_argo/workflow/relay/util/Fixture.java

```diff
@@ -20,6 +21,8 @@
       Class clazz, String fileName, Class<T> targetClass, ObjectMapper customMapper) {
     String json = loadJsonString(clazz, fileName);
     customMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    customMapper.registerModule(getModule());
```

Comment:
the only thing bothering me here is that as a static import it's not clear what the module is ... maybe `getOffsetDateTimeModule()`?